build.gradle

```diff
@@ -159,13 +169,15 @@ subprojects {
 
   check.dependsOn jacocoTestReport
 
+
   // Source jar
   // ----------
 
   task sourcesJar(type: Jar, dependsOn: classes) {
+    dependsOn generateProjectProperties
```

Comment:
Where is the generated file included - in -sources jar or in .class files jar? If it is in sources jar only, then it will not be accessible as a resource during runtime.

Reply:
.class files, because it's accessible when running tests or an Android APK.